### ISSUE TITLE
Add a --continue-from flag to run_with_beaker.py

### ISSUE THREAD

**Issue by matt-gardner** [allenai/allennlp#822]
Beaker isn't quite set up to handle this yet, but when they are, it'd be nice to have a flag that will let you continue training from a beaker experiment that failed due to memory or other issues.  You would pass in the dataset id as the argument to `--continue-from`, and the script would mount that as a source _and_ as the output directory, and would pass the `--recover` flag (from #861) to `python -m allennlp.run train`.

**Comment by matt-gardner** [comment]
It's actually a little bit more complicated than this, because you really should just grab the config dataset from the failed experiment, too...

**Comment by schmmd** [comment]
@ckarenz @splittingfield FYI.  Hopefully Matt has followed up with you guys separately about wanting to be able to continue an AllenNLP training session using the data from a past Beaker run.

**Comment by ckarenz** [comment]
Yes, we have several options here. I think the best and most likely option would be a (`--recover-from` or `--continue-from`) to pre-populate the experiment results previous run's before starting the container, then you'd pass the flag mentioned above.  I'll discuss with Matt offline and we can schedule any necessary work.

**Comment by schmmd** [comment]
This would also allow us to use spot instances ;-)

**Comment by matt-gardner** [comment]
I think we're taking a different path to this functionality, and having this issue open doesn't add anything.
